src/components/Token.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <g>
-    <circle cx="0" cy="0" r="1" :transform="`scale(${scale})`" :class="['player-token', 'planet-fill', planet]" />
+    <circle cx="0" cy="0" r="1.2" :transform="`scale(${scale})`" :class="['player-token', 'planet-fill', planet]" />
```

Comment:
Instead of modifying `r` here, it's better to modify `scale` in `ResearchTile.vue` (as the Token is also used to draw space stations)